docs/CODING_GUIDELINES_CORE.md

```diff
@@ -274,7 +376,7 @@ adhere to the following guidelines for our Core Web Component codebase.
 * We ship modern es2015 modules as the default.
 * Use a module bundler to dedupe and handle peer dependencies.
 * Share peer deps to prevent multiple versions running at one time.
-* `@clr/core/common` is private API do not use.
+* `@clr/core/common` is private internal API do not use.
```

Comment:
Should probably reword this. Its internal for core but a contributor can use it for new components internally and component consumers should not.